examples/dataset/semantic_segmentation/semantic_segmentation/upload_dataset.py

```diff
@@ -25,7 +25,7 @@
 def run(args: Namespace) -> None:
     df_dataset = pd.read_csv(args.dataset_csv, storage_options={"anon": True}, converters={"captions": pd.eval})
     df_dataset["mask"] = df_dataset["mask"].apply(lambda mask: SegmentationMask(locator=mask, labels={1: "PERSON"}))
-    upload_dataset(args.dataset_name, df_dataset)
+    upload_dataset(args.dataset_name, df_dataset, id_fields=["locator"])
```

Comment:
Needed to update to fix a separate breakage caused by https://github.com/kolenaIO/kolena/pull/546

Reply:
Before we land this — should we update the behavior to prefer `locator` when present?

Reply:
I would rather scope this change down to fixing the CI. Changing the `id_field` inference order is not backwards compatible
cc: @munkyshi

Reply:
Agreed, but from my perspective, we already broke it by changing it to prefer `*_id` or `id_*` over `locator`.

Reply:
Ugh, very true -- I hadn't really considered the breakage, sorry.

IMO preferring `*_id` is still the ideal end behavior (I'd prefer to go with an explicitly specified ID over assuming the locator uniquely qualifies a datapoint). What's the ID field that's taking precedence over `locator` in this case?